packages/util-package-extender/lib/js/_utils/_merge-packages.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * _merge-extended-package.js
+ * Get contents of package being extended
+ * Merge with contents of local package
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+const getRemoteFile = require('./_get-remote-file');
+
+/**
+ * Copy local file to outputDirectory
+ * @private
+ * @param {String} file name of the local file
+ * @param {String} sourcePath path to the local file
+ * @param {String} destinationPath output path of the file
+ * @return {Promise<Object>}
+ */
+async function mergeLocalFile(file, sourcePath, destinationPath) {
+	try {
+		await fs.copy(sourcePath, destinationPath);
```

Comment:
Not sure you want to but you can use `util.promisfy` to get promise versions of any callback based api. Save an extra install 

```
const fs = require('fs');
const {promisfy} = require('util');
const copy = promisify(fs.copyFile);
```

Reply:
You can! I always forget this 😄

Reply:
Just realised that I use `ensureDir` which isn't a standard node.fs command. Could rewrite it 🤔

Reply:
It's a fair bit more code to remove this dependency, so I'm leaning towards leaving it as is (remembering that this is why I didn't do this in the first place!)